### PR TITLE
Update java cookbook version lock

### PR DESCRIPTION
**Pull request by EMH333**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -18,9 +18,7 @@
 #
 
 # jenkins 1.612 and above requires java 7
-node.default['java']['jdk_version'] = '8'
-
-include_recipe 'java'
+openjdk_pkg_install '11'
```
Comment (ramereth) [inline recipes/default.rb:21]: Let's not upgrade this to 11 quite yet. Can we please keep this at 8 for now?

**Review by ramereth — CHANGES_REQUESTED** [review]

File: recipes/default.rb (hunk 1 above)
Comment (EMH333) [reply recipes/default.rb]: Sure, my bad!

File: metadata.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 depends          'base', '>= 2.6.0'
 depends          'osl-git'
-depends          'java', '< 8.0'
+depends          'java'
```
Comment (ramereth) [inline metadata.rb:13]: This is the first release which included `openjdk_pkg_install` so we should ensure we use that version.
```suggestion
depends          'java', '>= 8.1.0'
```

**Review by ramereth — CHANGES_REQUESTED** [review]
Can you please fix the new cookstyle issues and also update/add tests for this?

**Comment by ramereth** [comment]
@EMH333 so it looks like we should hold on doing this until #197 has been fixed and merged. I'm going to mark this as a draft for now.

**Comment by EMH333** [comment]
[Update to latest java cookbook](https://trello.com/c/xboLEOm1/2059-update-to-latest-java-cookbook)

**Review by ramereth — APPROVED** [review]
Please do a minor bump.

**Comment by EMH333** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v2.10.0.  Have a nice day!

**Pull request merged**